DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<p>Write the names of the data products with a different formatting, e.g., <code>Object</code> table.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>adjusted</p>